seg_lapa/train.py

```diff
@@ -149,7 +150,10 @@ def main(cfg: DictConfig):
     log_dir, run_id = create_log_dir(cfg)
 
     wb_logger = config.logger.get_logger(cfg, run_id, get_project_root())
-    model = DeeplabV3plus(config, log_dir)
+    if cfg.checkpoint_load.resume_training and cfg.checkpoint_load.checkpoints_filename is not None:
+        model = DeeplabV3plus.load_from_checkpoint(str(log_dir / cfg.checkpoint_load.checkpoints_filename), config=config, log_dir=log_dir)
```

Comment:
Since we're not using self.hparams, does this work? Are the values in the checkpoint overwritten by the values we pass in?